### PR TITLE
Monzo Pots

### PR DESCRIPTION
**Pull request by Sheaffy**
I have added the ability to view the currently authorised user, the ability to see the pots that they have, along with all their data.

### CODE REVIEW

**Comment by Sheaffy** [comment]
Just realised that someone else has also pulled in monzo pots.
I did however add pots to the API objects :)

**Comment by pawelad** [comment]
Awesome, thanks for the PR!

The CI failure is because of `flake8` (you can see the actual errors in Travis CI log) - could you please fix that? I'll also need the test for this endpoint, along with the [`vcrpy`](https://github.com/kevin1024/vcrpy) cassete before I merge this - you can either do that yourself (I'll be happy to help) or leave it to me (which will probably take a little bit more time), entirely up to you.

**Comment by Sheaffy** [comment]
Thats no problem I can do all of that..

**Comment by pawelad** [comment]
Awesome, just remember to redact your data from the cassete : )

**Comment by Sheaffy** [comment]
Okay, tox is now passing on flake8, will do the endpoint test now.

**Comment by Sheaffy** [comment]
@pawelad We should be all good to go now, a -0.1% decrease on coveralls though :(

File: src/pymonzo/monzo_api.py
Comment (pawelad) [inline src/pymonzo/monzo_api.py]: 100% a nitpick, but would you mind adding empty lines before `Official docs:` and `:param refresh:` so it's in the same format as other docstrings? Thanks!

**Review by pawelad — APPROVED** [review]
All this looks awesome, thank you!

**Comment by pawelad** [comment]
And sorry it took me almost a week - I was planning to deal with the coverage decrease as well but didn't find the time to do it and it shouldn't block this anyway. I'll try to open a PR for the couple conditional branches (`if not account_id` and `if not refresh`) on the weekend and do a proper release, possibly with [monz](https://github.com/pawelad/monz) update as well.

**Comment by pawelad** [comment]
Released in [v0.11.0](https://github.com/pawelad/pymonzo/releases/tag/v0.11.0)

**Pull request merged**
